package.json

```diff
@@ -8,7 +8,7 @@
     "build:l10njson": "pontoon-to-json --dest ./public",
     "build:exchangerates": "node scripts/exchangerates.js",
     "build:babel": "babel src --compact=true -s --out-dir dist",
-    "build:webpack": "webpack --optimize-minimize --optimize-dedupe",
+    "build:webpack": "webpack --optimize-minimize",
```

Comment:
webpack claimed to have removed `--optimize-dedupe` as an arg - I realize now I should check if it's deduping modules by default or not.

Reply:
They said it was redundant now, so my guess is they managed to get the deduping functionality, with 0 overhead.

Reply:
I couldn't find any documentation on how they made it redundant or what that means.